src/views/Incorporation/IncorporationAgreement.vue

```diff
@@ -207,7 +150,9 @@
 
       <section class="mt-10">
         <header id="confirm-agreement">
-          <h2>3. Confirm Incorporation Agreement and Article Completion</h2>
+          <h2 v-if="isTypeBcomp || isTypeBcCompany || isTypeBcCcc">3. Confirm Incorporation Agreement and Article
+            Completion</h2>
+          <h2 v-else-if="isTypeBcUlcCompany">2. Confirm Incorporation Agreement and Article Completion</h2>
         </header>
```

Comment:
Can you attach a screenshot of this page for BEN/BC/CC/ULC (and CP?) to this PR or the ticket, please?

Reply:
I have screenshots on the PR and I'll add them to this ticket aswell.